pallets/authors-manager/src/lib.rs

```diff
@@ -453,17 +456,7 @@ impl<T: Config> Pallet<T> {
 
         let author_id_bytes = <T as pallet::Config>::AccountToBytesConvert::into_bytes(author_id);
 
-        let function_name = b"removeAuthor";
-        let params = vec![
-            (b"bytes32".to_vec(), author_id_bytes.to_vec()),
-            (b"bytes".to_vec(), decompressed_eth_public_key.to_fixed_bytes().to_vec()),
-        ];
-        let tx_id = <T as pallet::Config>::BridgeInterface::publish(
-            function_name,
-            &params,
-            PALLET_ID.to_vec(),
-        )
-        .map_err(|e| DispatchError::Other(e.into()))?;
+        let tx_id = Self::publish_to_bridge(&decompressed_eth_public_key, &author_id_bytes)?;
```

Comment:
publish_to_bridge function always assumes `function_name = b"addAuthor";` We should take this as an argument.